### DISCUSSION TITLE
About high-capacity batteries

### DISCUSSION THREAD

**Discussion by wanghe0477** [egzumer/uv-k5-firmware-custom#335] [Q&A] [answered]
I buy a 3500mAh battery, can you add it to the menu options? I'm ashamed that I don't know how to code
![20231226_122342](https://github.com/egzumer/uv-k5-firmware-custom/assets/153917636/c5d9cd22-4298-4224-a8de-187d13f38c7e)
![20231226_122243](https://github.com/egzumer/uv-k5-firmware-custom/assets/153917636/6b2adcce-31cf-45b0-84f0-47bd8f64c3d6)

**Comment by pietss** [comment, marked as answer]
You can see here
https://github.com/egzumer/uv-k5-firmware-custom/issues/130
https://github.com/egzumer/uv-k5-firmware-custom/discussions/266

**Reply by wanghe0477** [reply]
Thank you
